File: src/main/java/ru/dragonestia/apocalypse/item/ScrapItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class ScrapItem extends Item implements AdvItem {

    public ScrapItem(Integer meta, int count) {
        super(ApocalypseID.SCRAP, meta, count, "Scrap");
    }

    @Override
    public String getRuName() {
        return "Металлолом";
    }

    @Override
    public String getIcon() {
        return "textures/items/scrap";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/ScrapItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class ScrapItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.SCRAP, 0, count, "Scrap"), AdvItem {

    override fun getRuName(): String = "Металлолом"

    override fun getIcon(): String = "textures/items/scrap"

}